### ISSUE TITLE
setting sensible 'defaults'?

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#763]
if the user doesnt have a line speed set in the config.. when they set a rate.. does it work/cause ill effects?

do we prompt user for line speed during wizard? if not.. we should.. 
or just set a default of something like 50 Mbps? that way setting the throttle works...

from irc:
```
[07:20pm] <wabbit> SABnzbd Version: 1.0.3
[07:20pm] <wabbit> Max Line Speed is blank
[07:21pm] <Cpuroast> ah
[07:21pm] <wabbit> I'll upgrade SAB..
[07:21pm] <Cpuroast> you have to populate the max line speed
[07:21pm] <wabbit> Didn't notice a new version
[07:21pm] <Cpuroast> for the percentage to work
[07:21pm] <wabbit> Ah ok
[07:21pm] <wabbit> Thank you!
[07:21pm] <Cpuroast> but yeah 1.1.1 is recommended
[07:21pm] <Cpuroast> 1.2.0 will make it a bit clearer
[07:21pm] <Cpuroast> it's currently in beta
[07:22pm] <wabbit> I loved the previous iteration of speed thorttle
[07:22pm] <wabbit> putting in 5000 for 5mb/sec
```

**Comment by Safihre** [comment]
We used to ask during wizard during 1.0.0 beta/alpha phase, but this turned out to cause confusion with Mb/MB. 
So we removed it from the wizard, so now when the user first starts SAB they can download at full speed and when they click to limit, they will see this:
![image](https://cloud.githubusercontent.com/assets/5703454/21637752/a581bca8-d268-11e6-8e9f-a11986dff784.png)

Which links to:
![image](https://cloud.githubusercontent.com/assets/5703454/21637641/2873bc3e-d268-11e6-9f02-bc83fa609935.png)
MB/s is selected by default.

Indeed like @Cpuroast says, we only added that kind of display in 1.1.1. 
In 1.0.3 users had to think more and it caused more confusion.
